### DISCUSSION TITLE
Improved cookbook/notes (Ingredients, meats, vegetables)

### DISCUSSION THREAD

**Discussion by legendaryelbone** [mealie-recipes/mealie#2699] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Providing more levels of depth to the app/cookbooks. Giving the ability to enhance ingredients, herbs, spices, meats, and vegetables. Being able to provide details, descriptions, notes.

### Please provide a concise description of the feature that would resolve your issue.

This started as an idea for an herbs and spices section or cook book. You have an herb called Basil, give a picture about it, description, what it tastes like and what it pairs well with. Then maybe a link for what it could be substituted for if you don't have Basil. On Discord while discussing this someone mentioned that we could do the same thing with meats and vegetables. Describe what the cut is or what cut is best for roasting, grilling ect.. 

![image](https://github.com/mealie-recipes/mealie/assets/149547666/daa9de2b-9800-4bcc-babf-13c6f1b89ce1)

This is where I got the idea starting on page 18 and 19
https://www.amazon.com/Herbs-Spices-reference-Jill-Norman/dp/B003R4ZH3Q?asin=1465435980&revisionId=&format=4&depth=1

A note section that we can add notes, columns, maybe a table.
Tables for portions, weights and measures, cooking times, temps for meats, methods of cooking, how to care for foods or tools. 

### Please consider and list out some caveats or tradeoffs made in your design decision

I know this is a recipe app but seeing the section called "Cookbooks" has given me the idea of including all these things listed above to make this a one stop app. This may not be for everyone or may be beyond the scope of this project.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by PureGeekery** [comment]
This is a feature I'm also interested in, although it seems like it was [denied in this discussion](https://github.com/mealie-recipes/mealie/discussions/1413). 

My inspiration was slightly different, I was inspired by the conversion chart in my physical recipe box and the magnet on my fridge with air-fry temperature and times. I think it could also be neat to pre-populate a few tables like this to get users started.

While I'm not a skilled developer myself, would be willing to contribute to the work for this in testing, documentation, and data entry (if shipping with some tables included).

**Reply by michael-genson** [reply]
I wouldn't say this was denied; that discussion was easily achievable using recipe notes for specific recipes. This discussion is notably different since it's related to cookbooks.

I believe Mealie used to have custom pages, but that was never a feature in 1.0 which is right around when I started using Mealie. I could see a use case for that in cookbooks.

**Reply by PureGeekery** [reply]
I think the reason I don't see it quite as something that fits within recipe notes, is because it's not necessarily something I am using with a recipe. Adding as a note also doesn't allow for recursion. If I find that 1 hour is a better cook time than the previously noted 45 minutes, I then have to update every recipe that has that note. Instead of changing it one place.

For example, I tend to cook chicken in bulk with my sous vide, and then use it in recipes throughout the week. I more need a quick reference. 

What could be an alternative solution, is to be able to have certain tags defaulted to "do not show" in the main recipe list. Then, you could create reference cookbook and link to the step with the particular information you need. Without also having to filter those items out of your main list every time.
